src/patternfly/components/Label/label.scss

```diff
@@ -1,31 +1,258 @@
 .pf-c-label {
   // Component
   --pf-c-label--PaddingTop: var(--pf-global--spacer--xs);
-  --pf-c-label--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-label--PaddingRight: var(--pf-global--spacer--md);
   --pf-c-label--PaddingBottom: var(--pf-global--spacer--xs);
-  --pf-c-label--PaddingLeft: var(--pf-global--spacer--sm);
-  --pf-c-label--BorderRadius: var(--pf-global--BorderRadius--sm);
-
-  // Color
-  --pf-c-label--BackgroundColor: var(--pf-global--primary-color--100);
-  --pf-c-label--Color: var(--pf-global--Color--light-100);
-
-  // Font
+  --pf-c-label--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-label--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-label--BackgroundColor: var(--pf-global--BackgroundColor--200);
+  --pf-c-label--Color: var(--pf-global--Color--100);
   --pf-c-label--FontSize: var(--pf-global--FontSize--sm);
 
-  // Modifiers
+  // Border base styles
+  --pf-c-label--before--BorderWidth: 0;
+  --pf-c-label--before--BorderColor: transparent;
+
+  // Compact
   --pf-c-label--m-compact--FontSize: var(--pf-global--FontSize--xs);
 
-  display: inline-block;
+  // Hoverable
+  --pf-c-label--m-hoverable--hover--before--BorderWidth: var(--pf-global--BorderWidth--sm);
```

Comment:
I left a comment but should this be called `selectable`

Reply:
After talking to @mcarrano the use case for these is as links. So I'm thinking of styling `a` and `button` this way and adding a `.pf-m-clickable` variation  to be used with a non-interactive element like `<div class="pf-c-label">some text<button class="pf-c-button pf-m-plain">x</button</div>` to create a closable/clickable label since you can't put a `<button>` in a `<button>` or `<a>` element. wdyt?

Reply:
ok that works!

Reply:
Sorry so much has changed in the last day! Since @maryshak1996 prefers the hover outline not be shown on a link label when you hover the close button, I separated the label into 2 children - a `__content` element (that can be a `<span>` or `<a>`) and the close button. So you just use an `<a>` on `__content` if you want a link, and the close button is an adjacent element (instead of a child), so no need for the `clickable` class manipulation.